src/Athens-Examples/AthensSurfaceExamples.class.st

```diff
@@ -615,11 +616,11 @@ AthensSurfaceExamples class >> exampleStrokeRect [
 			can pathTransform restoreAfter: [
 				can pathTransform rotateByDegrees: i*5.
 				can pathTransform scaleBy: (1- ( i/2000)).
-				surf clear.
-				can drawShape: (0@0 corner: 25@25) ].
-			Display getCanvas drawImage: 	surf asForm  at: 0@0. 
+				can drawShape: (0@0 corner: 25@25). ].
 		]	
-	]
+	].
+
```

Comment:
Could it be

`surf asMorph openInWindow`?

Or do we want that in general surfaces can be transformed into forms?